ddp-workspace/projects/ddp-dsm-ui/src/app/FON/fon.module.ts

```diff
@@ -68,7 +68,13 @@ export class fonModule {
     // Token
     const DSMToken = this.sessionService.getDSMToken();
     this.sessionService.setExpirationTime(DSMToken);
-    this.dssSessionService.setSession(null, DSMToken, null, LOCALE, +this.sessionService.getTokenExpiration());
+    this.dssSessionService.setSession(
+      null,
+      DSMToken,
+      null,
+      LOCALE,
+      +this.sessionService.getTokenExpiration()
+    );
```

Comment:
This is an example: just a code reformat with no functional change.
Please review your PR on github and make sure only thing that changes are your changes. And try to emulate existing style choices!